stratum/hal/lib/yang/yang_parse_tree.h

```diff
@@ -108,6 +111,14 @@ class TreeNode {
     return this;
   }
 
+  // Overrides the default-not-supported handler procedure called when
```

Comment:
`default-not-supported`?

Reply:
hmmmm this comes from copy-pasting the Stratum comment and method definition. We should get rid of this comment and re-write it for the special use-case that we have.

Reply:
Updated comment